exporters/stdout/stdoutmetric/exporter_test.go

```diff
@@ -97,7 +97,7 @@ func TestShutdownExporterReturnsShutdownErrorOnExport(t *testing.T) {
 	)
 	require.NoError(t, err)
 	require.NoError(t, exp.Shutdown(ctx))
-	assert.EqualError(t, exp.Export(ctx, data), "exporter shutdown")
+	assert.EqualError(t, exp.Export(ctx, &data), "exporter shutdown")
```

Comment:
I suggest changing the `data` variable initialization code (and add `&` or `new` there)

Reply:
updated as suggested